lib/rubocop/cop/minitest/no_focus.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Enforces tests are not focused
+      #
+      # @example
+      #   # bad
+      #   focus
+      #   test 'foo' do
+      #   end
+      #
+      #   # good
+      #   test 'foo' do
+      #   end
+      #
+      class NoFocus < Base
```

Comment:
Can you rename to `Minitest/Focus` similar to `RSpec/Focus`?

```suggestion
      class Focus < Base
```